sdk/storage/Azure.Storage.Common/src/Shared/StorageConnectionString.cs

```diff
@@ -66,6 +66,11 @@ internal class StorageConnectionString
         /// </summary>
         private static readonly AccountSetting s_fileEndpointSetting = Setting(Constants.ConnectionStrings.FileEndpointSetting, IsValidUri);
 
+        /// <summary>
```

Comment:
Is this file just a revert of previous changes?  Or does it need to be reviewed carefully?

Reply:
It should just be a revert of the table properties change, though I added a minor tweak to add defaults to the constructor Uri params.